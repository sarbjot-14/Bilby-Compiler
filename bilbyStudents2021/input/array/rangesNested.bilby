main{
	
	imm ranges := [[<2.4..5.6>,<3.3..9.8>],[<25.4..55.6>]];
	print ranges $n;
	
	imm i := [[<2..5>,<3..9>],[<22..52>,<32..92>]];
	print i $n;
	
	imm c := [[[<#a..#c>],[<#g..#o>]],[[<#d..#h>]]];
	print c $n;
	
	
	
}