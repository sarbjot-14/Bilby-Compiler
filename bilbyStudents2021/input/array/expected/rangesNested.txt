[[<2.400000..5.600000>, <3.300000..9.800000>], [<25.400000..55.600000>]]
[[<2..5>, <3..9>], [<22..52>, <32..92>]]
[[[<a..c>], [<g..o>]], [[<d..h>]]]
